src/test/resources/cql_valid_two/2015-04-01-13:58-change-waste-of-space-column-to-text.cql

```diff
@@ -1,2 +1,2 @@
 ALTER TABLE status DROP waste_of_space;
-ALTER TABLE status ADD waste_of_space text;
+ALTER TABLE status ADD waste_of_space varchar;
```

Comment:
Why did this change?

Reply:
so it seems that we cant change double to text any more. they re incompatible types. but we can change text to varchar so this way we kept the scenario of changing (compatible) types